Libraries/WebRTCUWP/InstallLibraries.ps1

```diff
@@ -124,5 +124,5 @@ Test-Nano()
             ($EditionId -eq "ServerTuva"))
 }
 
-DecompressZip -filename "Org.WebRtc_v1.1" -path "\Org.WebRTC\"
+DecompressZip -filename "Org.WebRtc_m58_v1.0" -path "\Org.WebRTC\"
```

Comment:
@anderm do we want this?

Reply:
That is fine, M58 library is stable at this point.